src/ValueObject/PUDO.php

```diff
@@ -12,6 +12,8 @@ class PUDO
     public bool $active;
     public Type $type;
     public string $language;
+    public ?string $name;
+    public ?int $ditance;
```

Comment:
Typo?

Reply:
What do you mean?
Brut XML is:
`<PUDO_ID>P66986</PUDO_ID>
            <DISTANCE>336</DISTANCE>
            <NAME>SAKADO</NAME>
            <ADDRESS1>8 rue Fecauderie</ADDRESS1>`

Reply:
ditance/distance

Reply:
Oh ok, it's fixed here : https://github.com/Snowbaha/dpd-pl-pickup-services-bundle/commit/d55142301ef0ed8dd3140812f00cc522ee86fd6d